xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/javascript.vm

```diff
@@ -271,11 +271,10 @@ $xwiki.jsfx.use("flamingo$jsExtension", {'forceSkinAction' : true, 'language' :
         #set($searchSuggestConfigHash = "${searchSuggestConfigHash}|${sourceId}")
       #end
     #end
-    $xwiki.jsfx.use('uicomponents/search/searchSuggest.js', {
-      'forceSkinAction' : true,
-      'h' : $searchSuggestConfigHash.hashCode()
-    })##
-    $xwiki.ssfx.use('uicomponents/search/searchSuggest.css', true)##
+    <script src="${services.webjars.url('org.xwiki.platform:xwiki-platform-search-webjar', 'searchSuggest.min.js', {'evaluate': true, 'h': $searchSuggestConfigHash.hashCode()})}" defer="defer"></script>
+    $xwiki.linkx.use(
+      $services.webjars.url('org.xwiki.platform:xwiki-platform-search-webjar', 'searchSuggest.min.css', {'evaluate': true}),
+      {'type': 'text/css', 'rel': 'stylesheet'})##
```

Comment:
Hmmm couldn't it be considered as a possible breakage? Isn't it possible that an extension used to use that searchSuggest.js or css?

Reply:
Although it's not documented as an API we could have extensions that manually load this JavaScript/CSS. Adding a mention in the release notes (in the migration section) should be enough IMO.